renovate.json

```diff
@@ -3,7 +3,7 @@
     "extends": [
       "config:base"
     ],
-    "prHourlyLimit": 4,
+    "prHourlyLimit": 1,
```

Comment:
What exactly is this hourly rate? One PR *per dependency*? Or across all of them?